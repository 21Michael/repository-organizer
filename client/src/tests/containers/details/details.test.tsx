import React from "react";
import { render, cleanup } from "@testing-library/react";
import Details from "../../../containers/details/details";
import "@testing-library/jest-dom/extend-expect";
import { BrowserRouter } from "react-router-dom";
import store from "../../../storeReduxToolkit/configStore";
import { Provider } from "react-redux";
import "mutationobserver-shim";

export default describe("Details: ", () => {
  afterEach(() => {
    cleanup();
  });

  test("details", () => {
    const { container } = render(
      <Provider store={store}>
        <BrowserRouter>
          <Details />
        </BrowserRouter>
      </Provider>
    );
    expect(container).toMatchSnapshot();
  });
});
