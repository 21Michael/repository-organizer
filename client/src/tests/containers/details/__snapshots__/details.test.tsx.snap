// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Details:  details 1`] = `
<div>
  <div
    class="wrapper"
  >
    <div
      class="details"
    >
      <div
        class="wrapper"
      >
        <h2
          class="title"
        >
          undefined details
        </h2>
        <ul
          class="list"
        />
      </div>
      <div
        class="button__wrapper"
      >
        <button
          class="button null"
          name="button"
          type="button"
        >
          back
        </button>
      </div>
    </div>
  </div>
</div>
`;
